docs/how-to/include_files.rst

```diff
@@ -0,0 +1,171 @@
+.. _how_to_include_files_and_resources:
+
+******************************************
+Including local files and remote resources
+******************************************
+
+Craft-parts provides built-in :ref:`dump plugin <craft_parts_dump_plugin>` for
+all kinds of projects that need to include local files and remote resources.
+This plugin is used to download, unpack, and copy files and directories from the
```

Comment:
Again, I think this emphasis on the dump plugin doing the downloading, unpacking, etc of the files is misleading because it suggests that the other plugins *don't* do that

Reply:
I've updated some of the wording and it should be clearer